src/VirtoCommerce.Platform.Web/wwwroot/js/app/modularity/modularity.js

```diff
@@ -129,10 +130,99 @@ angular.module('platformWebApp').config(['$stateProvider', function ($stateProvi
+
+                modulesApiMethod(selection, function (data) {
+                    blade.isLoading = false;
+
+                    var dialog = {
+                        id: "confirm",
+                        action: action,
+                        selection: selection,
+                        dependencies: data,
+                        callback: function (resume) {
+                            if (resume) {
+                                blade.isLoading = true;
+
+                                // combine selected modules with dependencies/dependents
+                                _.each(selection, function (x) {
+                                    if (!_.findWhere(data, { id: x.id })) {
+                                        data.push(x);
+                                    }
+                                });
+
+                                // choose the right API method
+                                switch (action) {
+                                    case 'install': modulesApiMethod = modulesApi.install;
+                                        break;
+                                    case 'update': modulesApiMethod = modulesApi.update;
+                                        break;
+                                    case 'uninstall': modulesApiMethod = modulesApi.uninstall;
+                                        break;
+                                }
+
+                                // apply the action
+                                modulesApiMethod(data, function (data) {
+                                    blade.isLoading = false;
+                                    // show module (un)installation progress
+                                    var newBlade = {
+                                        id: 'moduleInstallProgress',
+                                        currentEntity: data,
+                                        controller: 'platformWebApp.moduleInstallProgressController',
+                                        template: '$(Platform)/Scripts/app/modularity/wizards/newModule/module-wizard-progress-step.tpl.html'
+                                    };
+
+                                    // title depends on the action
+                                    switch (action) {
+                                        case 'install':_.extend(newBlade, { title: 'platform.blades.module-wizard-progress-step.title-install' });
+                                            break;
+                                        case 'update':_.extend(newBlade, { title: 'platform.blades.module-wizard-progress-step.title-update' });
+                                            break;
+                                        case 'uninstall':_.extend(newBlade, { title: 'platform.blades.module-wizard-progress-step.title-uninstall' });
+                                            break;
+                                    }
```

Comment:
can use method extracted above and newBlade.title = result.title

Reply:
fixed